components/dataset/ViewCard.vue

```diff
@@ -0,0 +1,118 @@
+<script setup lang="ts">
+interface Props {
+  view: string;
+  tableName: string;
+}
+
+const props = defineProps<Props>();
+const { t } = useI18n();
+
+// Get view display name
+const viewDisplayName = computed(() => {
+  return t(props.view) || props.view;
+});
+
+// Get view description based on type
+const viewDescription = computed(() => {
+  switch (props.view) {
+    case "map":
+      return "Borders, production sites, demographic changes.";
```

Comment:
```suggestion
      return "Geographic data visualized on a map";
```

However, internationalize